### PR TITLE
Piyush link free profile

### PR DESCRIPTION
**Pull request by Piyush142**
## Changes proposed

Added new json file.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3759"></a>

### CODE REVIEW

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please Rename the json file to `Piyush142.json`

**Comment by Piyush142** [comment]
> Please Rename the json file to `Piyush142.json`

Hi Chinmay, I have updated the file name.

**Review by schmelto — DISMISSED** [review]

File: data/Piyush142.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started OpenSource",
+      "date": "Jan 2023",
+      "icon": "FaGithub",
+      "color": "green",
+      "description": "Baby steps towards contributing",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (Panquesito7) [inline data/Piyush142.json:48]: I think you should update this link.

File: data/Piyush142.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    {
+      "title": "First Full Time Job",
+      "date": "Aug 2020",
+      "icon": "FaSuitcase",
+      "color": "green",
+      "description": "Full Stack Developer at Dunnhumby",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (Panquesito7) [inline data/Piyush142.json:56]: Same here.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Piyush142.json (hunk 1 above)
Comment (Piyush142) [reply data/Piyush142.json]: I have done the suggested changes.

File: data/Piyush142.json (hunk 2 above)
Comment (Piyush142) [reply data/Piyush142.json]: Done suggested changes.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 🎉

**Comment by Piyush142** [comment]
> Looks good to me! 🎉
Thank You!
Could you please provide review for requested changes. 🙏

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**